modules/tinymce/src/core/test/ts/browser/FormatterCheckTest.ts

```diff
@@ -163,6 +163,63 @@ describe('browser.tinymce.core.FormatterCheckTest', () => {
     assert.isTrue(editor.formatter.match('complex', { color: '#ff00' }), 'Selected element match with variable and function');
   });
 
+  context('using similar', () => {
+    const setupRegister = (editor: Editor) => {
```

Comment:
This should be a `before` hook, not called manually each time.

- [ ] Use mocha hooks instead of manually calling setup logic